### PR TITLE
Update configure-endpoints-mdm.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated Broken URl for windows 10 devices settings in intune.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6468

File: microsoft-365/security/defender-endpoint/configure-endpoints-mdm.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ For security reasons, the package used to Offboard devices will expire 30 days a
    - Date type: String
    - Value: [Copy and paste the value from the content of the WindowsDefenderATP_valid_until_YYYY-MM-DD.offboarding file]
 
-For more information on Microsoft Intune policy settings see, [Windows 10 policy settings in Microsoft Intune](/intune/deploy-use/windows-10-policy-settings-in-microsoft-intune).
+For more information on Microsoft Intune policy settings see, [Windows 10 policy settings in Microsoft Intune](/mem/intune/configuration/custom-settings-windows-10).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-mdm.md]:
```suggestion
For more information on Microsoft Intune policy settings, see [Windows 10 policy settings in Microsoft Intune](/mem/intune/configuration/custom-settings-windows-10).
```

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**